documentation/manual/helpers/OSHDBApplication.md

```diff
@@ -0,0 +1,98 @@
+# OSHDB Application Template
+## Installation
+
+Replace your OSHDB dependency with the following:
+
+```xml
+<dependency>
+   <groupId>org.heigit.ohsome</groupId>
+   <artifactId>oshdb-application-template</artifactId>
+   <version>0.7.2</version>
```

Comment:
Maybe better to replace this fix version with a sort of placeholder.

```suggestion
   <version>LATEST_VERSION</version>
```

Reply:
See comment above.